exotations/task_maps/task_map/include/task_map/JointAccBackFiniteDiff.h

```diff
@@ -0,0 +1,78 @@
+/*
+ *      Author: Christopher E. Mower
+ *
+ * Copyright (c) 2018, University Of Edinburgh
```

Comment:
Lower-case ``Of`` => ``of``

Reply:
Is this really necessary? As you can see below this is inconsistent with every other task map.
```
chris@chris:~/Projects/kuka_ws/src/exotica/exotations/task_maps/task_map/include/task_map$ grep -R University
Point2Line.hpp: * Copyright (c) 2018, University Of Edinburgh
CollisionCheck.h: * Copyright (c) 2017, University Of Edinburgh
Identity.h: * Copyright (c) 2017, University Of Edinburgh
JointVelBackFiniteDiff.h: * Copyright (c) 2018, University of Edinburgh
EffPosition.h: * Copyright (c) 2017, University Of Edinburgh
JointJerkBackFiniteDiff.h: * Copyright (c) 2018, University Of Edinburgh
CoM.h: * Copyright (c) 2017, University Of Edinburgh
JointAccBackFiniteDiff.h: * Copyright (c) 2018, University Of Edinburgh
EffFrame.h: * Copyright (c) 2017, University Of Edinburgh
JointLimit.h: * Copyright (c) 2017, University Of Edinburgh
EffOrientation.h: * Copyright (c) 2017, University Of Edinburgh
QuasiStatic.h: * Copyright (c) 2017, University Of Edinburgh
Distance.h: * Copyright (c) 2017, University Of Edinburgh
IMesh.h: * Copyright (c) 2017, University Of Edinburgh
SphereCollision.h: * Copyright (c) 2017, University Of Edinburgh
```